WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureOrchestrator.swift

```diff
@@ -0,0 +1,110 @@
+import Yosemite
+/// Orchestrates the sequence of actions required to capture a payment:
+/// 1. Check the there is a card reader connected
+/// 2. Launch the reader discovering and pairing UI if there is no reader connected
+/// 3. Obtain a Payment Intent from the card reader
```

Comment:
To make it easier for others coming across this and comparing it to https://stripe.com/docs/terminal/payments would you mind adding this in parentheses after #3:

(i.e., create a payment intent, collect a payment method, and process the payment)